lib/core/token_table.h

```diff
@@ -21,67 +21,69 @@
+    { { H2O_STRLIT("accept-language") }, 17, 0, 0, 0, 1 },
+    { { H2O_STRLIT("accept-ranges") }, 18, 0, 0, 0, 0 },
+    { { H2O_STRLIT("access-control-allow-origin") }, 20, 0, 0, 0, 0 },
+    { { H2O_STRLIT("age") }, 21, 0, 0, 0, 0 },
+    { { H2O_STRLIT("allow") }, 22, 0, 0, 0, 0 },
+    { { H2O_STRLIT("authorization") }, 23, 0, 0, 0, 0 },
+    { { H2O_STRLIT("cache-control") }, 24, 0, 0, 0, 0 },
+    { { H2O_STRLIT("cache-digest") }, 0, 0, 0, 0, 0 },
+    { { H2O_STRLIT("connection") }, 0, 1, 0, 1, 0 },
+    { { H2O_STRLIT("content-disposition") }, 25, 0, 0, 0, 0 },
+    { { H2O_STRLIT("content-encoding") }, 26, 0, 0, 0, 0 },
+    { { H2O_STRLIT("content-language") }, 27, 0, 0, 0, 0 },
+    { { H2O_STRLIT("content-length") }, 28, 0, 1, 0, 0 },
+    { { H2O_STRLIT("content-location") }, 29, 0, 0, 0, 0 },
+    { { H2O_STRLIT("content-range") }, 30, 0, 0, 0, 0 },
+    { { H2O_STRLIT("content-type") }, 31, 0, 0, 0, 0 },
+    { { H2O_STRLIT("cookie") }, 32, 0, 0, 0, 0 },
+    { { H2O_STRLIT("date") }, 33, 1, 0, 0, 0 },
+    { { H2O_STRLIT("etag") }, 34, 0, 0, 0, 0 },
+    { { H2O_STRLIT("expect") }, 35, 0, 1, 0, 0 },
+    { { H2O_STRLIT("expires") }, 36, 0, 0, 0, 0 },
+    { { H2O_STRLIT("from") }, 37, 0, 0, 0, 0 },
+    { { H2O_STRLIT("host") }, 38, 0, 1, 1, 0 },
+    { { H2O_STRLIT("http2-settings") }, 0, 1, 0, 1, 0 },
+    { { H2O_STRLIT("if-match") }, 39, 0, 0, 0, 0 },
+    { { H2O_STRLIT("if-modified-since") }, 40, 0, 0, 0, 0 },
+    { { H2O_STRLIT("if-none-match") }, 41, 0, 0, 0, 0 },
+    { { H2O_STRLIT("if-range") }, 42, 0, 0, 0, 0 },
+    { { H2O_STRLIT("if-unmodified-since") }, 43, 0, 0, 0, 0 },
+    { { H2O_STRLIT("keep-alive") }, 0, 1, 0, 0, 0 },
+    { { H2O_STRLIT("last-modified") }, 44, 0, 0, 0, 0 },
+    { { H2O_STRLIT("link") }, 45, 0, 0, 0, 0 },
+    { { H2O_STRLIT("location") }, 46, 0, 0, 0, 0 },
+    { { H2O_STRLIT("max-forwards") }, 47, 0, 0, 0, 0 },
+    { { H2O_STRLIT("proxy-authenticate") }, 48, 1, 0, 0, 0 },
+    { { H2O_STRLIT("proxy-authorization") }, 49, 1, 0, 0, 0 },
+    { { H2O_STRLIT("range") }, 50, 0, 0, 0, 0 },
+    { { H2O_STRLIT("referer") }, 51, 0, 0, 0, 0 },
+    { { H2O_STRLIT("refresh") }, 52, 0, 0, 0, 0 },
+    { { H2O_STRLIT("retry-after") }, 53, 0, 0, 0, 0 },
+    { { H2O_STRLIT("server") }, 54, 1, 0, 0, 0 },
+    { { H2O_STRLIT("set-cookie") }, 55, 0, 0, 0, 0 },
+    { { H2O_STRLIT("strict-transport-security") }, 56, 0, 0, 0, 0 },
+    { { H2O_STRLIT("te") }, 0, 1, 0, 1, 0 },
+    { { H2O_STRLIT("transfer-encoding") }, 57, 1, 1, 1, 0 },
+    { { H2O_STRLIT("upgrade") }, 0, 1, 1, 1, 0 },
+    { { H2O_STRLIT("user-agent") }, 58, 0, 0, 0, 1 },
+    { { H2O_STRLIT("vary") }, 59, 0, 0, 0, 0 },
+    { { H2O_STRLIT("via") }, 60, 0, 0, 0, 0 },
+    { { H2O_STRLIT("www-authenticate") }, 61, 0, 0, 0, 0 },
+    { { H2O_STRLIT("x-forwarded-for") }, 0, 0, 0, 0, 0 },
+    { { H2O_STRLIT("x-reproxy-url") }, 0, 0, 0, 0, 0 },
+    { { H2O_STRLIT("x-traffic") }, 0, 0, 0, 0, 0 }
+};
```

Comment:
Please completely revert the changes made to lib/core/token_table.h, so that the file would be left unchanged upon PR merger.